### PR TITLE
[#40] Ignore optional compile problems on *-gen source folders

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Comment by stadlerb** [comment]
Wouldn't "optional" be a better fit than "ignore_optional_problems"?

http://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2FIClasspathAttribute.html

IGNORE_OPTIONAL_PROBLEMS: If the value of this attribute is "true", all optional problems from the source folder described by this classpath entry will not be reported by the compiler.

OPTIONAL:  If the value of this attribute is "true", the classpath entry is optional. If the underlying resource or jar file doesn't exist, no error is reported and the classpath entry is ignored.

**Comment by kthoms** [comment]
No, it is correct that way.  The folders are not optional.

File: org.eclipse.xtend.core/.classpath [hunk 1]
```diff
@@ -27,6 +29,7 @@
 	</classpathentry>
 	<classpathentry kind="src" path="generator-xtend-gen">
```
Comment (LorenzoBettini) [inline org.eclipse.xtend.core/.classpath]: If I understand correctly this one is generated by Xtend, so it should not be "ignored" right?

Comment (kthoms) [inline org.eclipse.xtend.core/.classpath]: Right. Changed.

**Pull request merged**